File: Back/models/person.go
package models

import "time"

type Person struct {
	ID        uint `json:"id" gorm:"primaryKey"`
	CreatedAt time.Time
	FirstName string `json:"first_name"`
	LastName  string `json:"last_name"`
	Gender    string `json:"gender"`
	Phone     string `json:"phone"`
	Emails    string `json:"emails"`
	Addresses string `json:"addresses"`
	Age       int    `json:"age"`
	Birthday  string `json:"birthday"`
	Accounts  string `json:"accounts"`
}

File: Front/src/pages/lib/Types/Person.ts
import { AccountType } from ".";

export interface Person {
  id: number;
  first_name?: string;
  last_name?: string;
  emails?: string;
  gender?: string;
  phone?: string;
  addresses?: string;
  birthday?: string;
  age?: number;
  accounts: AccountType[];
}
